app/ItemList.java

```diff
@@ -0,0 +1,47 @@
+package icebase.app;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import icebase.icebase.Icebase;
+
+public class ItemList {
+    ItemList(){
+
+    }
+
+    public static void buyItem(String item){
+        System.out.println(item);
+        //display other item attributes
+    }
+    public static void main(String[] args){
+        Icebase ib = new Icebase();
+        Scanner sc = new Scanner(System.in);
+        int choice;
+        String storeName = "STORE X";
+        String categoryName = "CATEGORY X";
```

Comment:
I think dapat nga i pass in ni nga value diba? Like sa constructor

Reply:
right, naay empty constructor for that. wala pay gacreate sa instance tho so ako lang gi in-ana